mlir/test/Dialect/MemRef/load-store-alignment.mlir

```diff
@@ -0,0 +1,27 @@
+// RUN: mlir-opt -split-input-file -verify-diagnostics %s | FileCheck %s
```

Comment:
Please move the "positive" tests into `test/Dialect/*/ops.mlir` and negative tests into `test/Dialect/*/invalid.mlir`. Both exist in both dialects. We don't want to have a test file per operation.